addons/project/models/project.py

```diff
@@ -1817,6 +1817,14 @@ def stage_find(self, section_id, domain=[], order='sequence, id'):
         # perform search, return the first found
         return self.env['project.task.type'].search(search_domain, order=order, limit=1).id
 
+    @api.model
+    def get_sellable_line_domain(self, domain):
```

Comment:
Also, it is not needed to make that method public

```suggestion
    def _get_sellable_line_domain(self, domain):
```